src/docfx/build/resource/BuildResource.cs

```diff
@@ -22,6 +22,10 @@ internal static void Build(Context context, FilePath file)
             }
 
             context.PublishModelBuilder.SetPublishItem(file, metadata: null, outputPath);
+            if (isAdditionalResource)
+            {
+                context.PublishUrlMap.AddAdditionalResource(file);
```

Comment:
consider  let PublishModelBuilder call LinkResolver.GetAdditionalResources on building publish manifest?

With that this method doesn't need change

Reply:
This is feasible. It's just that `PublishModelBuilder` depending on `LinkResolver` feels a bit weird.